controllers/velero_test.go

```diff
@@ -1285,10 +1489,16 @@ func TestDPAReconciler_buildVeleroDeployment(t *testing.T) {
 			r := DPAReconciler{
 				Client: fakeClient,
 			}
-			if err := r.buildVeleroDeployment(tt.veleroDeployment, tt.dpa); (err != nil) != tt.wantErr {
+			err = nil
+			if err = r.buildVeleroDeployment(tt.veleroDeployment, tt.dpa); (err != nil) != tt.wantErr {
 				t.Errorf("buildVeleroDeployment() error = %v, wantErr %v", err, tt.wantErr)
 			}
+			if tt.wantErr && err != nil && tt.wantVeleroDeployment == nil {
+				// if we expect an error and we got one, and wantVeleroDeployment is not defined, we don't need to compare further.
+				t.Skip()
+			}
 			if !reflect.DeepEqual(tt.wantVeleroDeployment, tt.veleroDeployment) {
+				fmt.Println(cmp.Diff(tt.wantVeleroDeployment, tt.veleroDeployment))
```

Comment:
Please remove this line.

Reply:
addressed in https://github.com/openshift/oadp-operator/pull/588/commits/b90c45a496af5bfcf62d297c00e80bba68d71213